Make it possible to set "unknown" using __colanderalchemy_config__

elemoine opened the pull request:
This commit makes it possible to set the `unknown` argument passed to `colander.Mapping` using `__colanderalchemy_config__`. This is useful when one wants to set `unknown` (to `'raise'` or `'preserve'`) for classes used in a relationship.

Please tell me if more changes are required.

tisdall commented:
Nice!  Can you double check the docs to see if there's anywhere that change should be added in?  Also, please update the CHANGES.rst with a comment about the new change.

elemoine commented:
> Nice! Can you double check the docs to see if there's anywhere that change should be added in? Also, please update the CHANGES.rst with a comment about the new change.

Thanks for the review.

I updated CHANGES.rst and added a note to the customization doc page. The note is similar to what's in the docstrings of `SQLAlchemySchemaNode#init`, but I think it's worth having it there as well.

elemoine commented:
Thanks.

tisdall commented:
Thanks for the addition!
